fix: UTC-133: Password validation inconsistencies

nass600 opened the pull request:
Homogenizing password validation to 8-64 characters

<!--

This description MUST be filled out for a PR to receive a review. Its primary purposes are:

 - to enable your reviewer to review your code easily, and
 - to convince your reviewer that your code works as intended.

Some pointers to think about when filling out your PR description:
 - Reason for change: Description of problem and solution
 - Screenshots: All visible changes should include screenshots.
 - Rollout strategy: How will this code be rolled out? Feature flags / env var / other
 - Testing: Description of how this is being verified
 - Risks: Are there any known risks associated with this change, eg to security or performance?
 - Reviewer notes: Any info to help reviewers approve the PR
 - General notes: Any info to help onlookers understand the code, or callouts to significant portions.

You may use AI tools such as Copilot Actions to assist with writing your PR description (see https://docs.github.com/en/copilot/using-github-copilot/using-github-copilot-for-pull-requests/creating-a-pull-request-summary-with-github-copilot); however, an AI summary isn't enough by itself. You'll need to provide your reviewer with strong evidence that your code works as intended, which requires actually running the code and showing that it works.

-->

nass600 committed:
fix: UTC-133: Password validation inconsistencies

nass600 committed:
added missing file

nass600 committed:
fixed error messages

nass600 committed:
fixing blue issues

nass600 committed:
added validator

mcanu committed:
Increased limit to 128

mcanu commented:
/fmt
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/16009818929)

mcanu committed:
Apply pre-commit linters

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16009818929

mcanu commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/16009893355)

mcanu committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16009893355

mcanu committed:
Merge branch 'develop' into 'fb-utc-133'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16009893355

hakan458 commented on label_studio/users/validators.py:
```diff
@@ -0,0 +1,28 @@
+
+
+class PasswordLengthValidator:
+    def __init__(self, min_length=8, max_length=64):
```
should these defaults come from the settings as well?

hakan458 replied:
I see they are being passed, but these are not consistent

mcanu replied:
updated!

mcanu committed:
Addressed comment

mcanu commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/16028779423)

mcanu committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16028779423

mcanu committed:
Merge branch 'develop' into 'fb-utc-133'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16028779423
